portal/templates/activity/generate-activity-report.njk

```diff
@@ -0,0 +1,67 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% extends "index.njk" %}
+
+{% block pageTitle %}
+  View activity logs
+{% endblock %}
+
+{% block content %}
+  {{ govukBackLink({
+    href: "/",
+    attributes: { "data-cy": "back-link" }
+  }) }}
+
+  {% if errors %}
+  {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: errors.errorSummary,
+      attributes: {
+        'data-cy': 'error-summary'
+      }
+    }) }}
+{% endif %}
+
+  {% set headingHtml %}
+  <h1 class="govuk-heading-xl">
+    Which users activity should be featured in the report?
+  </h1>
+  <span class="govuk-caption-l govuk-!-margin-bottom-4">You can search with by name or email address </span>
```

Comment:
From https://design-system.service.gov.uk/components/text-input/, I think we should be using the `hint.text` option in `govukInput` for this